ext/memcached_rb/memcached_rb.h

```diff
@@ -2,17 +2,16 @@
 #define _RUBY_MEMCACHED_RB
 
 #include <ruby.h>
-
 #include <libmemcached/memcached.h>
-//#include <libmemcached/memcached_pool.h>
 
-typedef struct {
-  memcached_st * memc;
-} memcached_ctx;
+#define ARRAY_SIZE(a) (sizeof(a) / sizeof((a)[0]))
+#define UnwrapMemcached(rb, out) Data_Get_Struct(rb, memcached_st, out);
 
 extern VALUE rb_mMemcached;
+void rb_memcached_error_check(memcached_return_t rc);
 
-memcached_ctx * get_ctx(VALUE obj);
-void handle_memcached_return(memcached_return_t rc);
+void Init_memcached_rb_behavior(void);
```

Comment:
Merged the two header files in one because it was a *very* short header file :joy: